code/test/testYamlConversion.m

```diff
@@ -22,8 +22,12 @@
 
 
 % export to yml and then import back
-exportYaml(model,'testYamlConversion.yml');
-importedHumanGEM = importYaml('testYamlConversion.yml', true);
+try
+    exportYaml(model,'testYamlConversion.yml');
+    importedHumanGEM = importYaml('testYamlConversion.yml', true);
```

Comment:
I'm suggesting a change here to make sure the workflow fails in a way that is visible:

```suggestion
    importedHumanGEM = importYaml('testYamlConversionERROR.yml', true);
```

Reply:
This should obviously be reverted once we can confirm the workflow behaves as expected.

Reply:
Alright, it looks like the action still appears as passing when it should not.

Reply:
maybe add a new action file that can be a duplicate of `Test YAML conversion` for testing?

Reply:
> add a new action file

I'm not seeing how that would make any difference in the testing phase.